### PR TITLE
req: Generate UIDs with new prefixes

### PR DESCRIPTION
**Pull request by simhein**
Generate the UIDs for the specific sections with new prefixes.
Updated the linking of the new generated system UIDs in the software requirements.

### CODE REVIEW

File: docs/software_requirements/hw_arch_interface.sdoc [hunk 1]
```diff
@@ -1,12 +1,12 @@
 [DOCUMENT]
 TITLE: Hardware Architecture Interface
-REQ_PREFIX: ZEP-
+REQ_PREFIX: ZEP-SRS-14
```
Comment (Timotheous) [inline docs/software_requirements/hw_arch_interface.sdoc:3]: Prefix should end in a dash so the Requirements are uniquely formatted.  
Maintain the pattern from previous documents.

**Comment by Timotheous** [comment]
This looks good except for the missing "-" in the Hardware Architecture Interface REQ_PREFIX

This needs to be approved and merged as soon as possible to minimize the impact on requirements that are being written now.

File: docs/software_requirements/hw_arch_interface.sdoc (hunk 1 above)
Comment (simhein) [reply docs/software_requirements/hw_arch_interface.sdoc]: good catch thanks, fixed.

**Comment by simhein** [comment]
It seems there are no objections to go on with this. I will merge it to move forward and to minimize the impact of following PRs.

**Pull request merged**